_includes/featured_projects.html

```diff
@@ -1,3 +1,48 @@
-<div class="featured_projects">
 
+<!-- HTML Setup -->
+<div class="featured_projects">			
+					<h2 class="text-center text-white mt-3">MegaMinds at Work</h2>										
+		
+	
+	<!-- cards -->
```

Comment:
The file needs to be passed through an HTML linter for consistent spaces. Makes it easier to work with later